packages/cli/test/integration/shared/types.d.ts

```diff
@@ -15,9 +15,11 @@ export type SmtpTestAccount = {
 	};
 };
 
-export type EndpointNamespace = 'me' | 'users' | 'auth' | 'owner' | 'passwordReset' | 'credentials';
+type FunctionEndpointGroup = typeof FUNCTION_ENDPOINT_GROUP[number];
```

Comment:
Endpoint groups located in a function applied onto `App`. In future to become routers.

Renamed from `namespace` since neither functions nor routers are actual namespaces.